src/main/resources/messages/messages_nl.yml

```diff
@@ -40,7 +40,8 @@ error:
     max_registration: 'Je hebt het maximum aantal registraties overschreden (%reg_count/%max_acc: %reg_names).'
     logged_in: '&cJe bent al ingelogd!'
     kick_for_vip: '&cEen VIP-gebruiker heeft ingelogd toen de server vol was!'
-    # TODO kick_unresolved_hostname: '&cAn error occurred: unresolved player hostname!'
+    # TODO (Translator): Het onderste bericht is lastig te vertalen, moeten we 'computernaam' of 'hostname' gebruiken? Welke structuur kunnen wij het beste gebruiken voor 'unresolved'?
+    kick_unresolved_hostname: '&cEr heeft een fout plaatsgevonden: computernaam van de speler kon niet worden opgehaald!'
```

Comment:
Ik zou kiezen voor hostname. This is about not being able to resolve something from the user's IP address; computer name sounds like the name you can choose for your computer in the settings. But I'm not a native Dutch speaker.
opgehaald -> maybe opgelost? At least in German I would say something along the lines of "de hostnaam van de ip address kan niet worden opgelost"

Reply:
I changed it to hostname, as that is used by KPN (A major Dutch internet provider). https://www.internedservices.nl/faq/wat-vul-ik-in-bij-hostname/ . Furthermore, i changed `kon niet worden opgehaald!` to `kon niet gevonden worden!`, which litterally translates to `could not be found` instead of `could not be collected/picked up`. I think this is the best translation possible.

Ps. Het is `het ip adres` in plaats van `de ip address` :D

Reply:
Thanks for the correction :smile: I should have known both the spelling and the article... especially the spelling :D I need to use Dutch more again~